src/router_core/terminus.c

```diff
@@ -75,25 +76,47 @@ void qdr_terminus_free(qdr_terminus_t *term)
     free_qdr_terminus_t(term);
 }
 
+#ifdef TESTING
+__attribute__((weak))
```

Comment:
no need to make it weak since this goes to a shared library. __attribute__((noinline)) probably is a good idea